doc/developer-guides/hld/hv-startup.rst

```diff
@@ -185,69 +187,70 @@ The main steps include:
 
 SW configuration for Service VM (bzimage SW load as example):
 
--  **ACPI**: HV passes the entire ACPI table from bootloader to Service
+-  **ACPI**: HV passes the entire ACPI table from the bootloader to the Service
    VM directly. Legacy mode is currently supported as the ACPI table
    is loaded at F-Segment.
 
--  **E820**: HV passes e820 table from bootloader through zero-page
-   after the HV reserved (32M for example) and pre-launched VM owned
+-  **E820**: HV passes the e820 table from the bootloader through the zero page
+   after the HV reserved (32M, for example) and pre-launched User VM owned
```

Comment:
@dbkinder @fitchbe What does "after the HV reserved" mean -- seems like the sentence is missing a word?

Reply:
Yeah, not sure whether it should be "is reserved" or "reserved memory". (Or something else. :-)

Reply:
```suggestion
   after the HV reserved memory (32M, for example) and pre-launched User VM owned
```